__tests__/api.test.js

```diff
@@ -169,6 +169,51 @@ describe("POST API - insert a comment row using review_id and author FK", () =>
     })
 })
 
+describe("DELETE API - delete comment by id", () => {
+    test("200: ID exists - delete comment by id", () => {
+        return request(app)
+            .delete("/api/comments/2")
+            .expect(200)
+            .then(({body}) => {
+                const expected = {
+                    status: 200,
+                    command: 'DELETE',
+                    count: 1,
+                    note: 'Successfully deleted'
+                }
+                expect(body.deleted).toEqual(expected)
+            })
+    })
+    test("404: ID not exists: delete comment by id", () => {
+        return request(app)
+            .delete("/api/comments/20")
+            .expect(200)
+            .then(({body}) => {
+                const expected = {
+                    status: 404,
```

Comment:
in this test you are saying we expect a status of 200 - `.expect(200)` - but you are looking for a key of status in your response which holds a value of `404`

When express sends a status code via `res.status(<Status code here>)` that is information that is available to the client after making the request. So there is no need to manually create a property of `status` in your response object, that is doubling up on something that express has done for you